build/.goreleaser.yml

```diff
@@ -0,0 +1,106 @@
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
+    ignore:
+      - goos: darwin
+        goarch: 386
+
+  - id: nri-win
+    main: ./src
+    binary: nri-snmp
+    gobinary: "go1.9.7"
+    ldflags:
+      - -s -w -X main.integrationVersion={{.Version}} -X main.gitCommit={{.Commit}} -X main.buildDate={{.Date}}
+    goos:
+      - windows
+    goarch:
+      - 386
+      - amd64
+    hooks:
+      pre: build/windows/set_exe_properties.sh {{ .Env.TAG }} "snmp"
+
+nfpms:
+  - id: nri-deb
+    builds:
+      - nri-nix
+    file_name_template: "{{ .ProjectName }}_{{ .Version }}-1_{{ .Arch }}"
+    vendor: "New Relic, Inc."
+    homepage: "https://www.newrelic.com/infrastructure"
+    maintainer: "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
+    description: "New Relic Infrastructure snmp Integration extend the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from snmp components."
+    license: "https://newrelic.com/terms (also see LICENSE installed with this package)"
+    # Formats to be generated.
+    formats:
+      - deb
+    bindir: "/var/db/newrelic-infra/newrelic-integrations/bin"
+    files:
+      "snmp-config.yml.sample": "/etc/newrelic-infra/integrations.d/snmp-config.yml.sample"
+      "CHANGELOG.md": "/usr/share/doc/nri-snmp/CHANGELOG.md"
+      "README.md":  "/usr/share/doc/nri-snmp/README.md"
+      "LICENSE": "/usr/share/doc/nri-snmp/LICENSE"
+    config_files:
+      "snmp-definition.yml": "/var/db/newrelic-infra/newrelic-integrations/snmp-definition.yml"
+    dependencies:
+      - "newrelic-infra"
+
+  - id: nri-rpm
+    builds:
+      - nri-nix
+    file_name_template: "{{ .ProjectName }}-{{ .Version }}-{{ .Arch }}"
+    vendor: "New Relic, Inc."
+    homepage: "https://www.newrelic.com/infrastructure"
+    maintainer: "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
+    description: "New Relic Infrastructure snmp Integration extend the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from snmp components."
+    license: "https://newrelic.com/terms (also see LICENSE installed with this package)"
+    # Formats to be generated.
+    formats:
+      - rpm
+    bindir: "/var/db/newrelic-infra/newrelic-integrations/bin"
+    files:
+      "snmp-config.yml.sample": "/etc/newrelic-infra/integrations.d/snmp-config.yml.sample"
+      "CHANGELOG.md": "/usr/share/doc/nri-snmp/CHANGELOG.md"
+      "README.md":  "/usr/share/doc/nri-snmp/README.md"
+      "LICENSE": "/usr/share/doc/nri-snmp/LICENSE"
+    config_files:
+      "snmp-definition.yml": "/var/db/newrelic-infra/newrelic-integrations/snmp-definition.yml"
+    dependencies:
+      - "newrelic-infra"
+    overrides:
+      rpm:
+        replacements:
+          amd64: 1.x86_64
```

Comment:
What is the reason for adding `1.` to this replacement? This leads to `nri-mssql-0.0.1-1.x86_64.rpm`, as opposed to `nri-mssql-0.0.1-386.rpm` and `nri-mssql-0.0.1-arm.rpm`, which do not have a package release number.

Reply:
In the repo I see we use the following convention:
`nri-postgresql-2.6.1-1.x86_64.rpm`
So it would seem that are the other packages the ones who are wrong, and they don't seem to be uploaded to the repo.

Reply:
This is our legacy :( 
http://download.newrelic.com/infrastructure_agent/linux/yum/el/8/x86_64/

Reply:
Ok, let's keep it as it is then. Thanks!